### DISCUSSION TITLE
希望增加对国家哲学社会科学文献中心旗下文献的支持

### DISCUSSION THREAD

**Discussion by dingfeioutlook** [l0o0/jasminum#326] [Ideas]
今年以来，中国社会科学院旗下的不少期刊在知网上停止更新（包括《中国社会科学》、《经济研究》等），考虑到这些文献在中文期刊中的影响力，建议茉莉花插件增加对国家哲学社会科学文献中心旗下文献的支持。本人虽然不太懂Zotero插件的开发，不知道能通过何种方式做出一点贡献？谢谢！

**Comment by jiaojiaodubai** [comment]
浏览器抓取问题请到[Translators_CN](https://github.com/l0o0/translators_CN)反馈。

NSSD已经支持抓取，如果使用过程中有问题也到上述仓库反馈。
